app.js

```diff
@@ -131,3 +142,17 @@ async function triggerKaleidosPublications() {
     console.log(`Nothing to publish right now.`);
   }
 }
+
+async function retriggerFailedExportJobs() {
+  const failedJobs = await getFailedJobs();
+  console.debug(failedJobs);
+  for (let job of failedJobs) {
+    if (job.retryCount < config.export.job.maxRetryCount) {
```

Comment:
There is still the scenario of a job failing multiple times. In that case this job is silently ignored.
Perhaps put an `else` and log that job?
I assume that we want 0 failed jobs and 1 failed job would mean that there is a problem.

Reply:
:+1: 
We should also think about how we track _why_ the job failed. Currently the stacktrace will exist somewhere in the logs but that's cumbersome and we may only realize there's a repeatedly failing job very long after the fact (when the logs might have been overwritten). 

We don't do this elsewhere in the app though, so it's something we should think about and gradually apply everywhere (maybe store stacktraces as `?failedJob ext:failureEvent ?failureEvent . ?failureEvent ext:stackTrace """STACKTRACE""" ; ext:failedAt "timestamp" .`